server/src/main/java/org/apache/cassandra/sidecar/routes/sstableuploads/SSTableImportHandler.java

```diff
@@ -75,6 +82,13 @@ protected SSTableImportHandler(InstanceMetadataFetcher metadataFetcher,
         this.cache = cacheFactory.ssTableImportCache();
     }
 
+    @Override
+    public Set<Authorization> requiredAuthorizations()
+    {
+        List<String> eligibleResources = VariableAwareResource.DATA_WITH_KEYSPACE_TABLE.expandedResources();
```

Comment:
we are missing the CASSANDRA.MODIFY permission here

Reply:
Good catch updated this